### ISSUE TITLE
create CLI

### ISSUE THREAD

**Issue by suzuki-shunsuke** [suzuki-shunsuke/go-ci-env#11]
```
$ cienv --format [json|export]
```

```
$ cienv --formant export
export BRANCH=xxx
...
```

```
$ eval "$(cienv -f export)"
```

```
$ cienv
{
  "branch": "xxx"
}
```

**Comment by nopcoder** [comment]
@suzuki-shunsuke, did this meet your expectations? I've used `ci-env` instead of `cienv` to avoid conflicts with the existing folder named 'cienv'.

**Comment by suzuki-shunsuke** [comment]
Thank you for your contribution!
But sorry, this issue is very old (over four years ago) and I don't remember why this issue is still open.
I have already developed a similar tool.

https://github.com/suzuki-shunsuke/ci-info

ci-info was released about a month after this issue was created.

- This issue was created at Sep 10, 2020
- https://github.com/suzuki-shunsuke/ci-info/releases/tag/v0.1.0-0 (Oct 25, 2020)
